suse_caasp

```diff
@@ -263,7 +263,7 @@ docker_exec() {
         CONTAINERNAME=$2
         shift; shift;
         COMMAND="$@"
-        CONTAINER=$(docker ps | grep $CONTAINERNAME | awk '{print $1}')
+        CONTAINER=$(docker ps -a | grep $CONTAINERNAME | awk '{print $1}')
```

Comment:
There might be situation that result from command `$(docker ps -a | grep $CONTAINERNAME | awk '{print $1}')` will be multiple containers IDs. Couple lines below it will cause error

Reply:
For example:
```
caasp2-admin-lvfnhfnjh5dy:~ # docker ps -a | grep velum-dashboard | awk '{print $1}'
4164ee6068d7
2dfea20a46fc
```
```
echo $CONTAINER
4164ee6068d7 2dfea20a46fc
```
```
docker exec $CONTAINER logs
rpc error: code = 13 desc = invalid header field value "oci runtime error: exec failed: container_linux.go:247: starting container process caused \"exec: \\\"2dfea20a46fc\\\": executable file not found in $PATH\"\n"
```